pkg/cmd/server/start/start_master.go

```diff
@@ -563,21 +563,11 @@ func startControllers(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) erro
 			glog.Fatalf("Could not get client for HPA controller: %v", err)
 		}
 
-		_, _, recyclerClient, err := oc.GetServiceAccountClients(bootstrappolicy.InfraPersistentVolumeRecyclerControllerServiceAccountName)
```

Comment:
I'm going to make the recycler take a service account name argument, and
then move the permissions in.  The recycler is not exercised by any code in
our core path it appears.

On Sun, Jul 10, 2016 at 11:14 PM, Jordan Liggitt <notifications@github.com>
wrote:

> In pkg/cmd/server/start/start_master.go
> <https://github.com/openshift/origin/pull/9578#discussion_r70197548>:
>
> > @@ -563,21 +563,11 @@ func startControllers(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) erro
> >  			glog.Fatalf("Could not get client for HPA controller: %v", err)
> >  		}
> >
> > -		_, _, recyclerClient, err := oc.GetServiceAccountClients(bootstrappolicy.InfraPersistentVolumeRecyclerControllerServiceAccountName)
>
> the recycler SA has a custom role and gets added to the hostmount-anyuid
> SCC, which is required for recycling if I understand correctly. is the
> binder controller doing all of those things now? if so, it needs the
> appropriate role and scc
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9578/files/a4704975a98c7f4cfd15603a8a4b72be83872307..edbd37275083588ff64521b928f12f77d07c4c00#r70197548>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p9MB246RJx1mMuD6rqqNAVZL9K3-ks5qUbUigaJpZM4I-xTE>
> .
>